### DISCUSSION TITLE
General Questions

### DISCUSSION THREAD

**Discussion by terinfire** [notgiven688/jitterphysics2#103] [How do I ... ?]
Hello,

I currently am writing a (small-time commercial, already on Steam in Early Access) game engine/editor, with a physics backend that uses bepu2.  The performance is pretty nice, but I have a lot of issues with various things and I know Norbo is super limited on support/capacity.  How does Jitter2 compare to Bepu2 in performance?  Does it utilize some of the same ideas overall and vectorized collision/body processing?

Do you offer any ways to get "real time" support -- and/or is there a way to contact for potentially contracting to rip out what I have with bepu and/or build out what I need?  Loaded question, I realize -- and I intend to try to get some of it ported if I go this route first.  Just trying to see what can/not be done in terms of support.  Would be happy to discuss further in private.

Thank you in advance!

**Comment by notgiven688** [comment]
Hello, sorry for the late reply. 

>How does Jitter2 compare to Bepu2 in performance?

Jitter2 is very fast but not as performance orientated as Bepu. Bepu physics is just an awesome engine. You should see for yourself if Jitter fits your needs.

> Do you offer any ways to get "real time" support

This engine is a hobby project. Of course I am curious about your project. If you like we can discuss things further (jitterphysics at protonmail dot com).

**Reply by terinfire** [reply]
I probably owe you something (I'll send you an email) -- I ended up looking at a few of your demos and was able to get bepu working better.  The API/Framework for Jitter is a LOT easier to understand for Jitter.  Unfortunately, it's a large degree slower than bepu (for now).  I'll be watching in the future.
